lmdeploy/pytorch/backends/cuda/graph_runner.py

```diff
@@ -117,6 +121,20 @@ def __init__(self, model: torch.nn.Module, model_config: ModelConfig, cache_conf
         self.graph_pool_handle = torch.cuda.graph_pool_handle()
         self._runner_map: Dict[Any, CUDASingleGraphRunner] = dict()
 
+        self.compile_vit = False
+        torch_version = version.parse(torch.__version__)
+        if not self.backend_config.eager_mode and isinstance(
```

Comment:
Better not add model_special code in graphrunner. We can add a mixin in internvl to perform the condition and split batch.